any-sketch/src/main/cc/math/noise_parameters_computation.h

```diff
@@ -12,33 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef WFA_MEASUREMENT_COMMON_CRYPTO_NOISE_PARAMETERS_COMPUTATION_H_
-#define WFA_MEASUREMENT_COMMON_CRYPTO_NOISE_PARAMETERS_COMPUTATION_H_
+#ifndef SRC_MAIN_CC_MATH_NOISE_PARAMETERS_COMPUTATION_H_
+#define SRC_MAIN_CC_MATH_NOISE_PARAMETERS_COMPUTATION_H_
 
-#include "wfa/measurement/common/crypto/parameters.pb.h"
-#include "wfa/measurement/common/math/distributions.h"
+#include "math/distributions.h"
```

Comment:
It's weird to include wfa::common symbols from `wfa/common/`, but wfa::math symbols from `math/`.